requirements.txt

```diff
@@ -0,0 +1,5 @@
+codecov==2.0.15
```

Comment:
@dipanjanS the requirements.txt file has to be created by using `pip_compile` command on a manually created `requirements.in`. You can rename this file to `requirements.in` and then run `pip_compile` to auto-generate a `requirements.txt`.

Reply:
@dipanjanS looks good now.